### PR TITLE
Update resilience-with-device-states.md

### PR DESCRIPTION
**Pull request by ejahja**
Rebrand, “Windows Accounts extension" to “Microsoft Single Sign On”

### CODE REVIEW

File: docs/architecture/resilience-with-device-states.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ When a PRT requests access to an application, its device, session, and MFA claim
 ## How do I implement device states?
 
 * Enable [Microsoft Entra hybrid joined](~/identity/devices/hybrid-join-plan.md) and [Microsoft Entra join](~/identity/devices/device-join-plan.md) for company-owned Windows devices and require they be joined, if possible. If not possible, require they be registered. If there are older versions of Windows in your organization, upgrade those devices to use Windows 10.
-* Standardize user browser access to use either [Microsoft Edge](/deployedge/microsoft-edge-security-identity) or Google Chrome with the [Windows Accounts extension](https://chrome.google.com/webstore/detail/windows-10-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji) that enable seamless SSO to web applications using the PRT.
+* Standardize user browser access to use either [Microsoft Edge](/deployedge/microsoft-edge-security-identity) or Google Chrome with the [Microsoft Single Sign On](https://chrome.google.com/webstore/detail/windows-10-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji) that enable seamless SSO to web applications using the PRT.
```
Comment (iambmelt) [inline docs/architecture/resilience-with-device-states.md:28]:
```suggestion
* Standardize user browser access to use either [Microsoft Edge](/deployedge/microsoft-edge-security-identity) or Google Chrome with the [Microsoft Single Sign On extension](https://chrome.google.com/webstore/detail/windows-10-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji) that enable seamless SSO to web applications using the PRT.
```

**Review by iambmelt — APPROVED** [review]

File: docs/architecture/resilience-with-device-states.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ When a PRT requests access to an application, its device, session, and MFA claim
 ## How do I implement device states?
 
 * Enable [Microsoft Entra hybrid joined](~/identity/devices/hybrid-join-plan.md) and [Microsoft Entra join](~/identity/devices/device-join-plan.md) for company-owned Windows devices and require they be joined, if possible. If not possible, require they be registered. If there are older versions of Windows in your organization, upgrade those devices to use Windows 10.
-* Standardize user browser access to use either [Microsoft Edge](/deployedge/microsoft-edge-security-identity) or Google Chrome with the [Windows Accounts extension](https://chrome.google.com/webstore/detail/windows-10-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji) that enable seamless SSO to web applications using the PRT.
+* Standardize user browser access to use either [Microsoft Edge](/deployedge/microsoft-edge-security-identity) or Google Chrome with the [Microsoft Single Sign On extension](https://chrome.google.com/webstore/detail/windows-10-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji) that enable seamless SSO to web applications using the PRT.
```
Comment (EmLauber) [inline docs/architecture/resilience-with-device-states.md:28]:
```suggestion
* Standardize user browser access to use either [Microsoft Edge](/deployedge/microsoft-edge-security-identity) or Google Chrome with the [Microsoft Single Sign On extension](https://chrome.google.com/webstore/detail/windows-10-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji) (formerly known as the Windows Account extension) that enable seamless SSO to web applications using the PRT.
```

**Comment by Court72** [comment]
@janicericketts

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by janicericketts — APPROVED** [review]

**Comment by janicericketts** [comment]
#sign-off

**Pull request merged**